package.json

```diff
@@ -37,6 +43,7 @@
     "mini-css-extract-plugin": "^0.9.0",
     "mocha": "^7.0.1",
     "nyc": "^15.0.0",
+    "react-is": "^16.13.1",
```

Comment:
i think we don't need this

Reply:
this was added by the Storybook installation script. I removed it but it still gets included because it's a dependency of `@storybook/addon-action`.